packages/@react-types/color/src/index.d.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {
+  AriaLabelingProps,
+  AriaValidationProps,
+  FocusableDOMProps,
+  FocusableProps,
+  InputBase,
+  LabelableProps,
+  SpectrumLabelableProps,
+  StyleProps,
+  TextInputBase,
+  TextInputDOMProps,
+  Validation,
+} from '@react-types/shared';
+import {Color} from '@react-stately/color';
```

Comment:
Doing this felt kinda weird. Is this fine?

I was imagining `Color` from `react-stately` would extend an interface we define here in `react-types`, and that interface is also what we'll refer to as `Color` here in `HexColorFieldProps`.

I did this because I saw something similar from `@react-types/sidenav`
```
import {ReusableView} from '@react-stately/virtualizer';
```

Reply:
Yeah, we should make an interface for `Color` in `@react-types/color` and have the class in `@react-stately/color`  implement it.